tools/pipelines/test-real-service.yml

```diff
@@ -22,7 +22,6 @@ parameters:
   type: boolean
   default: false
 
-trigger: none
```

Comment:
What does removing this do?

Reply:
it will run as a CI build, so after every change